utils/utils.jsx

```diff
@@ -404,8 +404,13 @@ export function isGIFImage(extin) {
     return extin.toLowerCase() === Constants.IMAGE_TYPE_GIF;
 }
 
-export function getFileType(extin) {
-    var ext = extin.toLowerCase();
+const removeQuerystringOrHash = (extin) => {
+    return extin.split(/[?#]/)[0];
+};
+
+export const getFileType = (extin) => {
+    const ext = removeQuerystringOrHash(extin.toLowerCase());
```

Comment:
Shouldn't this be removed before we get to this function? It's not technically part of the file extension

Reply:
Fair enough